sample/src/main/java/com/chuckerteam/chucker/sample/MainActivity.kt

```diff
@@ -39,6 +40,7 @@ class MainActivity : AppCompatActivity() {
         )
 
         Chucker.registerDefaultCrashHandler(collector)
+        Chucker.setFeature(applicationContext, FeatureManager.Feature.HTTP_AND_ERROR)
```

Comment:
Here we should specify a list of features we want to enable. It's fair to think in the future there will be more features.
And by default, all the features will be enabled.

Something like :
```kotlin
Chucker.init(
    context = applicationContext,
    features = listOf(HttpFeature, ErrorFeature /* etc. */)
)
```

And it should be in the `Application` class.

ping. @cortinico @redwarp What do you think about it?